holobot/sdk/utils/exception_utils.py

```diff
@@ -35,11 +34,5 @@ def format_exception(exception: Exception) -> str:
     :rtype: str
     """
 
-    string_buffer = StringIO()
-    traceback.print_exception(exception, limit=None, file=string_buffer)
-    formatted_string = string_buffer.getvalue()
-    string_buffer.close()
-    if formatted_string[-1:] == "\n":
-        formatted_string = formatted_string[:-1]
-
-    return formatted_string
+    formatted_string = "".join(traceback.format_exception(exception))
+    return formatted_string.rstrip()
```

Comment:
This method is based on Python's built-in implementation for exception formatting and should be well optimized, so I'd prefer to leave it that way. (It's literally a shameless copy-paste because they don't expose it in an acceptable manner for this usage.)

Reply:
An additional argument to leave it the way it is is to be able to apply any changes if ever needed in the future without thinking.

Reply:
Since 3.10 `format_exception` matches `print_exception`'s output, but instead returns a list of string that should be concatenated.

Reply:
Do a quick benchmark please. If it's the same then it's okay.